docs/site/src/demos/tabs/tabs.md

```diff
@@ -31,3 +31,9 @@ Tab labels may be either all icons or all text.
 
 {{demo='demos/tabs/IconTabs.js'}}
 {{demo='demos/tabs/IconLabelTabs.js'}}
+
+## Disabled Tab
```

Comment:
I don't think that we need a new example section for that. We could add a disable variation in one of the previous examples.